model/package.json

```diff
@@ -28,40 +28,40 @@
   },
   "license": "SEE LICENSE IN LICENSE",
   "dependencies": {
-    "@babel/eslint-plugin": "^7.16.5",
-    "@babel/runtime": "^7.17.2",
     "boom": "7.3.0",
     "btoa": "^1.2.1",
-    "depth-first": "^4.0.0",
-    "eslint-plugin-import": "^2.25.4",
-    "expr-eval": "2.0.2",
     "govuk-frontend": "^4.0.1",
-    "hmpo-components": "5.2.1",
-    "joi": "17.2.1",
-    "nanoid": "^3.3.1",
-    "nunjucks": "3.2.1",
-    "path": "0.12.7",
-    "wreck": "14.2.0"
+    "joi": "17.2.1"
   },
   "devDependencies": {
     "@babel/cli": "^7.10.5",
     "@babel/core": "^7.11.1",
     "@babel/eslint-parser": "^7.17.0",
+    "@babel/eslint-plugin": "^7.16.5",
     "@babel/plugin-proposal-class-properties": "7.16.7",
     "@babel/plugin-proposal-export-default-from": "^7.16.7",
     "@babel/plugin-proposal-private-methods": "^7.16.11",
     "@babel/plugin-transform-runtime": "^7.17.0",
     "@babel/preset-env": "^7.16.11",
     "@babel/preset-typescript": "^7.16.7",
+    "@babel/runtime": "^7.17.2",
```

Comment:
babel-runtime needs to be a main dep https://babeljs.io/docs/en/babel-runtime#usage

Reply:
Moved back to dependencies